lmn/models.py

```diff
@@ -46,11 +46,19 @@ def __str__(self):
 
 ''' One user's opinion of one show. '''
 class Note(models.Model):
+    RATINGS = (
```

Comment:
I merged a change to the model make sure to resolve the conflict on the conversations page.

Reply:
Will do. Thanks, homie!